modular_skyrat/modules/modular_items/code/psych_cart.dm

```diff
@@ -0,0 +1,12 @@
+/obj/item/cartridge/psychologist
+	name = "\improper P.A.T cartridge"
+	desc = "Flaunt that Psychology Degree with this Cartridge!"
+	icon_state = "cart-m"
+	access = CART_MEDICAL
+	spam_enabled = 1
+
+/obj/item/pda/psychologist
+	name = "Psychologist PDA"
```

Comment:
Are all PDAs capitalized in such a fashion? I got a feeling the job name shouldn't be capitalized here.